SAMtags.tex

```diff
@@ -247,8 +247,21 @@ \subsection{Metadata}
 \end{description}
 
 \subsection{Barcodes}
+DNA Barcodes can be used to identify the provenance of the underlying the reads. 
+There are currently several usages of barcodes that may co-exist: Sample Barcode, Cell Barcode, and Molecular-barcode.
```

Comment:
usages -> applications

Also, I found a forth variant:
1. Molecular Barcode
2. Molecular barcode
3. molecular barcode
4. molecular-barcode
!

Reply:
Too many "the": "the underlying the reads".

Reply:
Or “varieties” — _applications_ leaves the possibility that you could chose to use BC for all of the different applications…